test/benchmarks/material/button/app.module.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import {A11yModule} from '@angular/cdk/a11y';
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {MatButtonModule} from '@angular/material/button';
+
+/**
+ * @title Button benchmark component.
+ */
+@Component({
+  selector: 'app-root',
+  template: `
+    <button id="show" (click)="show()">Show</button>
+    <button id="hide" (click)="hide()">Hide</button>
+
+    <ng-container *ngIf="isVisible">
+			<button mat-raised-button>Basic</button>
+    </ng-container>
+  `,
+  encapsulation: ViewEncapsulation.None,
+  styleUrls: ['//src/material/core/theming/prebuilt/indigo-pink.css'],
+})
+export class ButtonBenchmarkApp {
+  isChecked = false;
+  isVisible = false;
+
+  show() { this.isVisible = true; }
+  hide() { this.isVisible = false; }
+}
+
+
+@NgModule({
+  declarations: [ButtonBenchmarkApp],
+  imports: [
+    A11yModule,
```

Comment:
Consider removing `A11yModule`. I know I'm repeating this from other PRs, but it's just for tracking